Cpanel/Tests/Cpanel/Core/AbstractTest.php

```diff
@@ -55,7 +55,7 @@ public function getCS($methods = array(), $args = array(), $mockName = '', $call
         if (empty($methods)) {
             $methods = null;
         }
-        $m = $this->getMock($this->cut, $methods, $args, $mockName, $callConst, $callClone, $callA);
+        $m = $this->_makeMock($this->cut, $methods, $args, $mockName, $callConst, $callClone, $callA);
```

Comment:
What made us go from a public to private(-ish) method?

Reply:
getMock is no longer a directly supported function. Rather than replacing all instances of it with a complicated chain to get what we wanted anyway, I abstracted it into the protected _makeMock function.